### PR TITLE
fix tests for JuliaDiff/ChainRulesCore.jl#298

### PR DESCRIPTION
**Pull request by simeonschaub**

### CODE REVIEW

File: test/rulesets/Random/random.jl [hunk 1]
```diff
@@ -46,7 +46,7 @@ Random.rand(d::NormalDistribution) = d.μ + d.σ*randn()
             x, pb = rrule(rand, args...)
             @test x isa xType
             dself, dargs = Iterators.peel(pb(10.0))
-            @test dself isa Zero
+            @test dself == DoesNotExist()
```
Comment (oxinabox) [inline test/rulesets/Random/random.jl:49]: Maybe just
```suggestion
            @test iszero(dself)
```

**Review by oxinabox — APPROVED** [review]

File: Project.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 name = "ChainRules"
 uuid = "082447d4-558c-5d27-93f4-14fc19e9eca2"
-version = "0.7.50"
+version = "0.7.51"
```
Comment (oxinabox) [inline Project.toml:3]: this doesn't require a version bump as it isn't user visable.
But a bump is permitted

Comment (simeonschaub) [reply Project.toml:3]: Yeah, you are right, it isn't necessary here.
```suggestion
version = "0.7.50"
```

**Comment by simeonschaub** [comment]
Nightly failures are just #356

**Pull request merged**